tests/build_/test_build_request.py

```diff
@@ -24,7 +24,6 @@
 
 from flexmock import flexmock
 import pytest
-from pkg_resources import parse_version
```

Comment:
No, it's actually currently imported twice, that's just removing the second redundant import.